src/config.inc

```diff
@@ -72,3 +72,6 @@ OSX_IDENTITY="Developer ID Application: Daniel Kroening"
 
 # Detailed version information
 CBMC_VERSION = 5.11
+
+# Use the CUDD library for BDDs, can be installed using `make -C src cudd-download`
```

Comment:
maybe needs documentation in COMPILING.md as well

Reply:
added